apps/engage-hub-service/src/communities/dto/update-community.dto.ts

```diff
@@ -0,0 +1,31 @@
+import { IsEnum, IsOptional, IsNumber, IsString, IsUrl } from 'class-validator';
+import { SpecializationTypes } from 'src/trainer-request/entities/especialization-types.entity';
+
+export class UpdateCommunityDto {
+  @IsString()
+  @IsOptional()
+  name: string;
```

Comment:
It is recommended to use ? and @IsOptional() together on optional properties of a DTO: ? tells TypeScript that they are not mandatory, and @IsOptional() prevents errors in class-validator if they are absent. 

Example:
@IsString()
@IsOptional()
name?: string;

Reply:
It has been fixed in new commit, please review it.